README.md

```diff
@@ -17,52 +20,74 @@ Zeppelin is a moderation bot for Discord, designed with large servers and reliab
 - Starboard
 - And more!
 
-See https://zeppelin.gg/ for more details.
+See <https://zeppelin.gg/> for more details.
 
 ## Development
-These instructions are intended for bot development only.
+
+These instructions are intended for bot development only, they are not recommended for self-hosting.
 
 👉 **No support is offered for self-hosting the bot!** 👈
 
-### Running the bot
-1. `cd backend`
-2. `npm ci`
-3. Make a copy of `bot.env.example` called `bot.env`, fill in the values
-4. Run the desired start script:
-    * `npm run build` followed by `npm run start-bot-dev` to run the bot in a **development** environment
-    * `npm run build` followed by `npm run start-bot-prod` to run the bot in a **production** environment
-    * `npm run watch` to watch files and run the **bot and api both** in a **development** environment
-      with automatic restart on file changes
-5. When testing, make sure you have your test server in the `allowed_guilds` table or the guild's config won't be loaded at all
-
-### Running the API server
-1. `cd backend`
-2. `npm ci`
-3. Make a copy of `api.env.example` called `api.env`, fill in the values
-4. Run the desired start script:
-    * `npm run build` followed by `npm run start-api-dev` to run the api in a **development** environment
-    * `npm run build` followed by `npm run start-api-prod` to run the api in a **production** environment
-    * `npm run watch` to watch files and run the **bot and api both** in a **development** environment
-      with automatic restart on file changes
+### Running the backend
+
+1. Go into the backend directory: `cd backend`
+
+2. Install dependencies: `npm ci`
+
+3. Make a copy of `bot.env.example` and `api.env.example` (removing the `.example` suffix), fill in the values.
+  There are defaults for your convenience, feel free to replace these.
+
+4. Setup the database: `npm run migrate-dev`
+
+5. To start the backend, there are two alternatives:
+    - **It is recommended** to use `npm run watch` for development. This will automatically restart on save
+    - Run `npm run build` followed by the desired start script:
+      - `npm run start-bot-dev` to start the bot.
+      - `start-api-dev` to start the api server.
+
+6. On the first run the bot will attempt to add all IDs from `STAFF` in `api.env` into the database
```

Comment:
```suggestion
1. Go into the backend directory: `cd backend`

1. Install dependencies: `npm ci`

1. Make a copy of `bot.env.example` and `api.env.example` (removing the `.example` suffix), fill in the values.
  There are defaults for your convenience, feel free to replace these.

1. Setup the database: `npm run migrate-dev`

1. To start the backend, there are two alternatives:
    - **It is recommended** to use `npm run watch` for development. This will automatically restart on save
    - Run `npm run build` followed by the desired start script:
      - `npm run start-bot-dev` to start the bot.
      - `start-api-dev` to start the api server.

1. On the first run the bot will attempt to add all IDs from `STAFF` in `api.env` into the database
```